Automattic/jetpack
# VideoPress: Add register_videopress_blocks() method to register all VideoPress blocks

This new method `register_videopress_blocks()` register all VideoPress blocks.

Fixes #

#### Changes proposed in this Pull Request:

*  VideoPress: Add register_videopress_blocks() method to register all VideoPress blocks

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






The method is not used yet, so the idea is to apply the changes and confirm the app keeps working as expected. No breaking changes.
